posts/solving-parallel-letter-frequency-elixir-part-1-unicode-match-regex.md

```diff
@@ -1,6 +1,8 @@
-# "Solving Parallel Letter Frequency in Elixir - Part I: Unicode Matching in Elixir"
+# 1. What Parallel Letter Frequency can teach you about Unicode matching in Elixir
```

Comment:
Let's get rid of the number here, as it won't make sense outside of the context of a series.